src/components/gameScreen/components/answerItem.js

```diff
@@ -1,34 +1,40 @@
 import React, {useState} from 'react';
-import {Text, TouchableOpacity, StyleSheet, View} from 'react-native';
-import {scoreStore} from '../../../store/scoreStore';
+import {Text, TouchableOpacity, StyleSheet} from 'react-native';
+import {observer} from 'mobx-react-lite';
+import scoreStore from '../../../store/scoreStore';
+import store from '../../../store/store';
 
-const rightAnswerColor = '#b2d092';
-const wrongAnswerColor = '#ff6c6c';
+const rightAnswerColor = '#cae1b0';
+const wrongAnswerColor = 'rgba(245,133,133,0.64)';
 
-export const AnswerItem = ({
+export const AnswerItem = observer(({
   answer,
   handleNextQuestion,
   currentRightAnswer,
 }) => {
   const [answerBackground, setAnswerBackground] = useState('#fff');
-
   const setAnswerItemBackground = () => {
-    if (answer === currentRightAnswer) {
-      setAnswerBackground(rightAnswerColor);
-      scoreStore.setScore(scoreStore.score + 1);
-      setTimeout(() => handleNextQuestion(false), 500);
-    } else {
-      setAnswerBackground(wrongAnswerColor);
-      setTimeout(() => handleNextQuestion(false), 500);
+    if (!store.isAnswerHasChosen) {
+      store.setIsAnswerHasChosen(true);
+      if (answer === currentRightAnswer) {
+        setAnswerBackground(rightAnswerColor);
+        scoreStore.setScore(scoreStore.score + 1);
+        setTimeout(() => handleNextQuestion(false), 500);
+      } else {
+        setAnswerBackground(wrongAnswerColor);
+        setTimeout(() => handleNextQuestion(false), 500);
```

Comment:
duplicated code, you can move this line out of if block and out of else block and just write 1 line below if else block